app/javascript/src/styles/_custom.scss

```diff
@@ -0,0 +1,5 @@
+// Remove left dropdown menu oddities
+.ml-auto .dropdown-menu {
+  left: auto !important;
```

Comment:
lets not use `!important`. Use specificity instead. The more specific your css is, the higher its rank is. Therefore it will supersede other lower scored rules.

does this make sense?

Reply:
Yes po, I'll definitely look for an alternative and follow proper specificity solutions but for now this would be a temporary _patch_ as I need to override these predefined Bootstrap classes to remove odd spacing for the dropdown menu items.